clang/lib/CConv/AVarBoundsInfo.cpp

```diff
@@ -10,13 +10,21 @@
 //
```

Comment:
Seems like you completely changed this file. I'll look at it from scratch. Can you say at a high level what it does (I think it does the propagation/inference process) and what its entry point is? Then I will start looking at it from that entry point.

Reply:
Will do.